extensions/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java

```diff
@@ -69,4 +72,16 @@ private boolean isHtmlFileName(String fileName) {
         return fileName.endsWith(".html") || fileName.endsWith(".htm");
     }
 
+    @Record(STATIC_INIT)
+    @BuildStep(onlyIf = IsDevelopment.class)
+    void addAdditionalEndpointsExceptionMapper(List<NotFoundPageDisplayableAdditionalBuildItem> displayableAdditionalBuildItems,
+            ExceptionMapperRecorder recorder, HttpRootPathBuildItem httpRoot) {
+        Set<String> endpoints = displayableAdditionalBuildItems
+                .stream()
+                .map(displayableAdditionalBuildItem -> httpRoot.adjustPath(displayableAdditionalBuildItem.getEndpoint())
+                        .substring(1))
+                .collect(Collectors.toSet());
```

Comment:
Why collect this into a `Set` here when it is later converted to a `List` within `NotFoundExceptionMapper`? We could collect this straight into a sorted list here

Reply:
done

Reply:
I don't see this change, perhaps you did not include it in the commit?

Reply:
I changed the type to Set here https://github.com/quarkusio/quarkus/pull/5235/files#diff-21cab9c4a8e7fb3b9a63321deb01676bR48

Reply:
Oh! Ok, I didn't expect this. But it's ok then.

Reply:
I'm not sure passing a `SortedSet` implementation to the recorder and the mapper is a good idea compared with a simple `List`. I suspect the `set` has a higher resources consumption while we don't need to sort the elements or eliminate duplicates at runtime. WDYT?

Reply:
Ah I missed this. 

You mean change the `set` to `list`, I am sure this can be done, I was more learning in the usage of `set` to eliminate duplicates but this is handled nicely by the http layer. We can keep the sort though, so that endpoints are ordered correctly. WDYT?